pkg/quota/admission/resourcequota/admission.go

```diff
@@ -54,7 +54,7 @@ func (a *originQuotaAdmission) Admit(as admission.Attributes) error {
 }
 
 func (a *originQuotaAdmission) SetOpenshiftClient(osClient osclient.Interface) {
-	registry := quota.NewRegistry(osClient)
+	registry := quota.NewRegistry(osClient, true)
```

Comment:
@derekwaynecarr - I get the feeling we need some sort of lifecycle method that can be called now that we have setters.  If the controller needs to set up reflectors or anything else that relies on the OpenShift client it's probably going to happen here for now and kind of makes the method jumbled.